src/Table/Table.js

```diff
@@ -69,7 +69,7 @@ Table.propTypes = {
 };
 
 Table.propDescriptions = {
-    tableData: 'Array of objects that contain two properties: `rowData` (an array of strings containing data for each column in the row), and `children` (an array of objects containing additional rows).',
+    tableData: 'Array of objects that contain one property: `rowData` (an array of strings containing data for each column in the row).',
```

Comment:
Not quite sure where this previous description came from, but that is _NOT_ how the component works.

Reply:
Array of objects _contains_ right?

Reply:
Actually, I think it is just _contain_ because "objects" is plural.